en/micro-integrator/docs/setup/deployment/deploying_wso2_ei.md

```diff
@@ -150,6 +150,9 @@ When you have scheduled tasks in your integration deployment, each task should o
     [[task_resolver]]
     task_server_count = "3"
     ```
+    
+!!!Note
+task_server_count applies each time a task is resolved. i.e when a node leaves the cluster, the task will be resolved (assgined) to another node only if the task_server_count is satisfied.
```

Comment:
Shall we say "task re-assignment gets delayed until the task server count is satisfied"? Otherwise it'll mean like task scheduling is forever stopped

Reply:
yes +1